functions/tips-scopes/main.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import time
+
+
+# Placeholder
+def heavy_computation():
+    return time.time()
+
+
+# Placeholder
+def light_computation():
+    return time.time()
+
+
+# [START functions_tips_scopes]
+# Global (instance-wide) scope
+# This computation runs at instance cold-start
+instance_var = heavy_computation()
+
+
+def scope_demo(request):
```

Comment:
Duplicated from tips PR

Reply:
I assume you're referring to [this sample](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/8554/files#diff-325f55f73ee26cd4a1d3c3d4fe5b3005ab41b3451c9f7b09cd98c29f9ac6f73bR29-R57)?

They're similar, but not identical - that sample [caches the "local" variable globally](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/8554/files#diff-325f55f73ee26cd4a1d3c3d4fe5b3005ab41b3451c9f7b09cd98c29f9ac6f73bR49-R53), while this sample does not.